### ISSUE TITLE
Add `.enabled` to configuration

### ISSUE THREAD

**Issue by tomajwinter** [Schwad/chat_gpt_error_handler#2]
Hi @Schwad 👋.

This is really nice - thanks for your work 🙌.

I just went to raise a PR to sketch out this idea but had a permissions error (please shout if the repo should be accepting PRs & I've likely done something wrong with my token)

I'd like to add the gem to a development environment whilst allowing other folks to opt in / out. My understanding currently is if I add the gem to our app but omit the `OPENAI_ACCESS_TOKEN` it errors on [this line](https://github.com/Schwad/chat_gpt_error_handler/blob/master/lib/chat_gpt_error_handler/error_handler.rb#L9). If I put a dummy string it'll error [on this line](https://github.com/Schwad/chat_gpt_error_handler/blob/master/lib/chat_gpt_error_handler/error_handler.rb#L46)

I was wondering about allowing something like this in my app's initializer

```
ChatGptErrorHandler.openai_access_token = ENV.fetch('OPENAI_ACCESS_TOKEN', nil)
ChatGptErrorHandler.enabled = ENV.fetch('CHAT_GPT_ERROR_HANDLER', false)
```

then in `ChatGptErrorHandler::ErrorHandler`

```
# lib/chat_gpt_error_handler
module ChatGptErrorHandler
    mattr_accessor :openai_access_token, :enabled

  # Default value for the openai_access_token, used when not explicitly set by the user
  @@openai_access_token = nil

  # Default value to allow opting-in
  @@enabled = false
end

# lib/chat_gpt_error_handler/error_handler.rb
require 'openai'

module ChatGptErrorHandler
  class ErrorHandler
    def initialize(app)
      @app = app
      if ChatGptErrorHandler.enabled
        OpenAI.configure do |config|
          config.access_token = ChatGptErrorHandler.openai_access_token || ENV.fetch('OPENAI_ACCESS_TOKEN')
        end
      end
    end

    def call(env)
      begin
        @app.call(env)
      rescue => error
        send_to_gpt(error) if ChatGptErrorHandler.enabled
        raise error
      end
    end

    private

   ...
```

What do you think? Not sure on the best API or if there's a better pattern to `require: false` the gem and then wrap the initializer in a conditional but I couldn't get something like that to work easily in our app

Thanks again 🙌

**Comment by Schwad** [comment]
I think this makes sense. Thanks for proposing this! Should it be able to work if you say, fork the repo, make the changes, then open a pull request against that? Unless I too am missing something, I don't own many gems folks would be willing to contribute to :)

**Comment by tomajwinter** [comment]
Thanks @Schwad! Made an effort at the fork PR here https://github.com/Schwad/chat_gpt_error_handler/pull/3

**Comment by Schwad** [comment]
Fixed by #3
